contributed_definitions/nyaml/NXmpes_xps_liquid.yaml

```diff
@@ -0,0 +1,314 @@
+#Pincelli, Rettig, Arora at fhi-berlin.mpg.de, Dobener at hu-berlin.de, 06/2022
+#Draft version of a NeXus application definition for photoemission,
+#It is designed to be extended by other application definitions
+#with higher granularity in the data description.
+
+doc: This is the application definition for multidimensional photoelectron spectroscopy on liquids
+category: application
+NXmpes:
+  ENTRY(NXentry):
```

Comment:
Writing ENTRY is not necessary. It is automatically inferred from writing `(NXentry)`, i.e., there is always a conversion `(NXsomething)` -> `SOMETHING(NXsomething)` for groups.